workbench/src/renderer/components/SetupTab/index.jsx

```diff
@@ -397,11 +398,20 @@ class SetupTab extends React.Component {
         argsEnabled: await fetchArgsEnabled({
           model_module: pyModuleName,
           args: JSON.stringify(argsDictFromObject(argsValues)),
-        })
+        }),
       });
     }
   }
 
+  debouncedDropdownFunctions() {

```

Comment:
It doesn't look like this debounced version is being used. And we'll need to remember to clean it up in `componentWillUnmount`

Reply:
Oh good catch